db/migrate/20210917040713_create_general_metrics.rb

```diff
@@ -0,0 +1,12 @@
+class CreateGeneralMetrics < ActiveRecord::Migration[6.1]
+  def change
+    create_table :general_metrics do |t|
+      t.integer :distance
+      t.timestamp :movingTime
+      t.integer :elevationGain
```

Comment:
Ruby typically has a snake_case convention, so instead of `elevationGain`, use `elevation_gain`